### PR TITLE
Adds permissions for transformer

### PR DESCRIPTION
**Pull request by kenoir**
Publishing to SNS & reading from kinesis stream.

In order to run in prod!

### CODE REVIEW

**Comment by kenoir** [comment]
Worth noting this IAM file is probably due for a cleanup / modularisation.

File: terraform/iam.tf [hunk 1]
```diff
@@ -51,3 +51,41 @@ data "aws_iam_policy_document" "read_ingestor_q" {
     ]
   }
 }
+
+resource "aws_iam_role_policy" "ecs_transformer_task_sns" {
+  name = "ecs_task_jenkins_policy"
+  role = "${module.transformer.role_name}"
+
+  policy = "${data.aws_iam_policy_document.publish_to_calm_sns.json}"
```
Comment (alexwlchan) [inline terraform/iam.tf]: `terraform fmt` please.

File: terraform/iam.tf [hunk 2]
```diff
@@ -51,3 +51,41 @@ data "aws_iam_policy_document" "read_ingestor_q" {
...
+  name = "ecs_task_jenkins_policy"
+  role = "${module.transformer.role_name}"
+
+  policy = "${data.aws_iam_policy_document.read_calm_kinesis_stream.json}"
+}
+
+data "aws_iam_policy_document" "read_calm_kinesis_stream" {
+  statement {
+    actions = [
+      "dynamodb:*",
```
Comment (alexwlchan) [inline terraform/iam.tf]: Ditto comment on previous PR about scoping Dynamo permissions more tightly.

Comment (kenoir) [inline terraform/iam.tf]: There isn't really a smaller usable scope for streams: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Operations_Amazon_DynamoDB_Streams.html

File: terraform/iam.tf (hunk 1 above)
Comment (kenoir) [inline terraform/iam.tf]: errrrr, what is problem?

**Pull request merged**